PROJECT

```diff
@@ -0,0 +1,13 @@
+domain: nodefeaturediscoveries.nfd.kubernetes.io
+layout: go.kubebuilder.io/v3
+projectName: node-feature-discovery
+repo: github.com/kubernetes-sigs/node-feature-discovery-operator
+resources:
+- crdVersion: v1
+  group: nfd.kubernetes.io
+  kind: NodeFeatureDiscovery
+  version: v1
+version: 3-alpha
```

Comment:
stupid question: where does this version come from, what does it refer to?

Reply:
ping

Reply:
This is a stanza for the operator framework, that `version` is not pointing to NFD version
https://sdk.operatorframework.io/docs/cli/operator-sdk_alpha_config-3alpha-to-3/